ETHCHAT.md

```diff
@@ -46,21 +48,26 @@ This protocol will consist of several stages:
 
 ## Considerations on the X3DH initialization
 This scheme requires working on specific elliptic curves which differ from those used by Ethereum. 
-To be precise, Ethereum makes use of the curve secp256k1, whereas X3DH requires either X25519 or X448. For the sake of simplicity we will assume the selected curve is X25519.
+To be precise, Ethereum makes use of the curve secp256k1, whereas X3DH requires either X25519 or X448. For security reasons one must work on the curve X448.
 
 Bob and Alice must define a key pair (ik, IK) where:
 -   The key ik must be kept secret,
 -   and the key IK is public.
 
 Bob will not be able to use his Ethereum public key during this stage due to incompatibilities with the involved elliptic curves, therefore it will be required to generate new keys. 
-This can be done using the basepoint $G$ for X25519 and $ik \in \mathbb{Z}_p$ a random integer:
+This can be done using the basepoint $G$ for X448 and $ik \in \mathbb{Z}_p$ a random integer:
 
 $$ IK = ik \cdot G $$
 
 The scheme X3DH will also require the generation of a public key SPK which will be generated repeating the above process: one takes $spk \in \mathbb{Z}_p$ a secret random integer and computes:
 
 $$ SPK = spk \cdot G $$
 
+SPK is a public key generated and stored at medium-term. It is called a signed prekey because Bob also needs to store a public key certificate of SPK using IK. 
```

Comment:
```suggestion
SPK is a public key generated and stored at medium-term. 
It is called a signed prekey because Bob also needs to store a public key certificate of SPK using IK. 
```